be/src/exprs/agg/aggregate.h

```diff
@@ -282,7 +283,12 @@ class AggregateFunctionStateHelper : public AggregateFunction {
 public:
     static constexpr bool pod_state() { return std::is_trivially_destructible_v<State>; }
 
-    void create(FunctionContext* ctx, AggDataPtr __restrict ptr) const final { new (ptr) State; }
+    void create(FunctionContext* ctx, AggDataPtr __restrict ptr) const final {
+        new (ptr) State;
+        init(ctx, ptr);
```

Comment:
we shouldn't change create method

Reply:
here need that init() after create aggregation functions.